frontend/internal-packages/artifact/src/schemas/artifact.ts

```diff
@@ -9,7 +9,6 @@ export const dmlExecutionLogSchema = v.object({
 
 // DML operation schema
 export const dmlOperationSchema = v.object({
-  testCaseId: v.string(),
```

Comment:
I think it's probably better to remove the testCaseId from the seed's dmlOperation as well.

https://github.com/liam-hq/liam/blob/0ce501dce174cf06bec3379841594753a082796f/frontend/internal-packages/db/supabase/seed.sql#L556-L558